libdnf.spec

```diff
@@ -121,14 +136,17 @@ pushd build-py3
 popd
 %endif
 
+# Mageia uses file triggers for this instead
+%if 0%{?mageia} == 0
 %post -p /sbin/ldconfig
 %postun -p /sbin/ldconfig
+%endif
 
 %files
 %license COPYING
 %doc README.md AUTHORS NEWS
-%{_libdir}/%{name}.so.*
-%{_libdir}/girepository-1.0/Dnf-*.typelib
+%{_libdir}/%{name}.so.%{sover}
```

Comment:
It forces it to match with the provides that I added. And it's a hard enforcer to ensure you know about your ABI, I guess. IMO, it's a good thing.